citi-onboarding/tenis-center
# feature(components-file):added components directory

## What I did

- Create directory for components

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/components-file`
   1. `git pull origin feature/components-file`

1.Check if directory is created and already has a index 

---

**Time spent:** <see your time spent on https://wakatime.com/dashboard >
